contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6293,9 +6328,13 @@ transformSelectIntoStmt(CreateTableAsStmt *stmt)
 	}
 
 	result = lappend(result, stmt);
```

Comment:
My initial comment was to move this line up in the top and directly append everything else into `result` instead of creating new lists.

Reply:
it won't work, as altstmt should be appended before our column list. Altstmt create the identity column and our alterTablestmt(newstmt, which stores the original name) should be appended after Altstmt (the alterTablestmt which creates the identity column).